db/import_mb_data.py

```diff
@@ -248,117 +179,53 @@ def load_end_area_type(connection, gids_in_AB):
             ON area.id = artist.end_area
     INNER JOIN artist_credit
             ON artist.id = artist_credit.id
-    INNER JOIN recording
-            ON artist_credit.id = recording.artist_credit
-         WHERE recording.gid in :gids
+         WHERE artist_credit.id in :data
     """)
-    result = connection.execute(end_area_type_query, {'gids': tuple(gids_in_AB)})
+    result = connection.execute(end_area_type_query, {'data': tuple(artist_credit_from_recording)})
     MB_end_area_type_data = result.fetchall()
 
     return MB_end_area_type_data
 
 
-def load_release_status(connection, gids_in_AB, MB_release_data):
+def load_release_status(connection):
     """Fetch release_status table data from MusicBrainz database for the
     recording MBIDs in AcousticBrainz database.
 
     Args:
         connection: database connection to execute the query.
-        MB_release_data (of type - sqlalchemy.resultproxy): data retrieved from the release table
-            of the MusicBrainz database (should contain release_status values).
     Returns:
         release_status data fetched from MusicBrainz database.
     """
-    filters = []
-    filter_data = {}
-
-    # Get data corresponding to status column in release table
-    MB_release_fk_status = list({value['status'] for value in MB_release_data})
-
-    if gids_in_AB:
-        filters.append("recording.gid in :gids")
-        filter_data["gids"] = tuple(gids_in_AB)
-
-    if MB_release_data:
-        filters.append("release_status.id in :data")
-        filter_data["data"] = tuple(MB_release_fk_status)
-
-    filterstr = " OR ".join(filters)
-    if filterstr:
-        filterstr = " WHERE " + filterstr
-
     release_status_query = text("""
-        SELECT DISTINCT release_status.id,
-               release_status.name,
-               release_status.parent,
-               release_status.child_order,
-               release_status.description,
-               release_status.gid
-          FROM release_status
-    INNER JOIN release
-            ON release.status = release_status.id
-        INNER JOIN recording
-            ON recording.artist_credit = release.artist_credit
-            {filterstr}
-    """.format(filterstr=filterstr)
-    )
-    result = connection.execute(release_status_query, filter_data)
+        SELECT * FROM release_status
+             ORDER BY id
+    """)
+    result = connection.execute(release_status_query)
     MB_release_status_data = result.fetchall()
 
     return MB_release_status_data
 
 
-def load_release_group_primary_type(connection, gids_in_AB, MB_release_group_data):
+def load_release_group_primary_type(connection):
     """Fetch release_group_primary_type table data from MusicBrainz database for the
```

Comment:
now that these functions are fetching entire tables, we should change docs to reflect that.